eng/WpfArcadeSdk/tools/CodeGen/AvTrace/GenTraceSources.targets

```diff
@@ -0,0 +1,26 @@
+<Project>
+  <PropertyGroup>
+    <WindowsBaseAvTraceMessageFile>$(WpfSourceDir)\WindowsBase\AvTraceMessages.json</WindowsBaseAvTraceMessageFile>
+    <PresentationCoreAvTraceMessageFile>$(WpfSourceDir)\PresentationCore\AvTraceMessages.json</PresentationCoreAvTraceMessageFile>
+    <PresentationFrameworkAvTraceMessageFile>$(WpfSourceDir)\PresentationFramework\AvTraceMessages.json</PresentationFrameworkAvTraceMessageFile>    
+  </PropertyGroup>
+
+  <ItemGroup>
+    <None Include="$(MSBuildThisFileDirectory)PresentationTraceSources.tt">
+      <Generator>TextTemplatingFileGenerator</Generator>
+      <LastGenOutput>PresentationTraceSources.cs</LastGenOutput>
+      <OutputFilePath>$(WpfSourceDir)\WindowsBase\Generated\</OutputFilePath>
```

Comment:
This is a design change - instead of generating ephemeral sources into `$(IntermediateOutputPath)`, now you are generating permanent sources under `$(WpfSourceDir)`. This seems consistent with the way TT is supposed to be used. This should be called out in the change-summary in both this PR and the internal one that corresponds.

Reply:
Yeah this is one of the things i especially like about this :)